sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql

```diff
@@ -1,13 +1,13 @@
 SELECT
   DATE(submission_timestamp) AS submission_date,
-  REGEXP_EXTRACT(_TABLE_SUFFIX, "(?:stub_)?(?:[^_]+_)(.*)__") AS dataset_id,
+  REPLACE(metadata.document_namespace, '-', '_') AS dataset_id,
   COUNT(*) AS num_rows,
 FROM
-  `payload_bytes_decoded.*`
+  `payload_bytes_decoded_all`
```

Comment:
I was wondering why this works without table is unqualified.

Reply:
It needs to be qualified.